docs/Index.tsx

```diff
@@ -217,19 +198,19 @@ class MemberEditView extends React.Component<IMemberEditViewProps, IMemberEditVi
   render() {
     return (
       <PopoverComponent
+        ref='popover'
         attachment='top left'
         targetAttachment='bottom left'
         constraints={[{
           to: 'scrollParent',
           pin: true
         }]}
-        toggleOpenedTetherElement={this.props.toggleOpenedTetherElement}
         >
-        <button className='btn' onClick={() => this.props.toggleOpenedTetherElement(!this.props.isOpen)}>
+        <button className='btn'>
           {this.props.memberModel.isNew() ? 'Add member' : this.props.memberModel.email}
         </button>
         {
-          this.props.isOpen && this.getPopoverContent()
+          this.getPopoverContent()
```

Comment:
I simplified it a little and removed the this.getPopoverContent method.. was not really useful anyway.